projects/1.20-fabric/assets/more-wizards/morewizardsmod/lang/zh_cn.json

```
@@ -0,0 +1,41 @@
+{
+  "item.morewizardsmod.more_wizards_lightning_spell_book": "雷电魔法书",
+  "item.morewizardsmod.more_wizards_soul_spell_book": "灵魂魔法书",
+  "item.morewizardsmod.lightning_staff": "雷电权杖",
+  "item.morewizardsmod.netherite_lightning_staff": "下界合金雷电权杖",
+  "item.morewizardsmod.soul_staff": "灵魂权杖",
+  "item.morewizardsmod.netherite_soul_staff": "下界合金灵魂权杖",
+
+  "item.morewizardsmod.lightning_wizard_armor_t1_head": "雷电学徒帽",
+  "item.morewizardsmod.lightning_wizard_armor_t1_chest": "雷电学徒长袍 ",
+  "item.morewizardsmod.lightning_wizard_armor_t1_legs": "雷电学徒护腿",
+  "item.morewizardsmod.lightning_wizard_armor_t1_feet": "雷电学徒靴",
+  "item.morewizardsmod.lightning_wizard_armor_t2_head": "雷电魔法师帽",
+  "item.morewizardsmod.lightning_wizard_armor_t2_chest": "雷电魔法师长袍",
+  "item.morewizardsmod.lightning_wizard_armor_t2_legs": "雷电魔法师护腿",
+  "item.morewizardsmod.lightning_wizard_armor_t2_feet": "雷电魔法师靴",
+
+  "item.morewizardsmod.soul_wizard_armor_t1_head": "灵魂学徒帽",
+  "item.morewizardsmod.soul_wizard_armor_t1_chest": "灵魂学徒长袍",
+  "item.morewizardsmod.soul_wizard_armor_t1_legs": "灵魂学徒护腿",
+  "item.morewizardsmod.soul_wizard_armor_t1_feet": "灵魂学徒靴",
+  "item.morewizardsmod.soul_wizard_armor_t2_head": "灵魂魔法师帽",
+  "item.morewizardsmod.soul_wizard_armor_t2_chest": "灵魂魔法师长袍",
+  "item.morewizardsmod.soul_wizard_armor_t2_legs": "灵魂魔法师护腿",
+  "item.morewizardsmod.soul_wizard_armor_t2_feet": "灵魂魔法师靴",
+
+  "spell.morewizardsmod.lightning_strike.name": "雷霆一击",
+  "spell.morewizardsmod.lightning_strike.description": "蓄力3秒后选中并电击一只目标，造成伤害",

```

```suggestion
  "spell.morewizardsmod.lightning_strike.description": "召唤一束闪电束并电击一个敌方目标3次",
```
原文：`Strike an enemy target with a lightning bolt that hits 3 times`

确定原文的X times是蓄力，而不是击中次数吗
以下的这些description都有诸如此类的错译问题，请全部修改

---

感谢指正，实测下来原文的x times是击中次数。
这个闪电并非原版闪电，第一次打死后闪电能自动寻找第二个目标。